airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -667,7 +667,7 @@ def state(self) -> Optional[Mapping[str, Any]]:
 
     @state.setter
     def state(self, value):
-        state_value = value[self.updated_at_field]
+        state_value = value.get(self.updated_at_field, self._state)
```

Comment:
we should create an issue to refactor state management in this connector. It's very confusing that we have: 

`get_updated_state`
`_update_state`
`@state.setter`
`_start_date` and `_state` always set to the same thing

Reply:
Created https://github.com/airbytehq/airbyte/issues/11392